apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveStakingFlow.tsx

```diff
@@ -16,6 +17,21 @@ import perFamilyManageActions from "~/renderer/generated/AccountHeaderManageActi
 
 export const LOCAL_STORAGE_KEY_PREFIX = "receive_staking_";
 
+export const CheckBoxContainer: ThemedComponent<{ state: string }> = styled(Flex)`
+  & > div {
+    column-gap: 15px;
```

Comment:
Not to be changed in this PR as it's just reshuffling existing components, but I think it would be nicer if we could change how this div looked by passing props to the child component... for example;

```
<CheckBoxContainer p={6}>
   <CheckBox columnGap="15px" />
</CheckBoxContainer>
```

I think that's a more idiomatic way of styling react components as it means we don't need to know the internals of what `CheckBox` has. Maybe something to keep in mind for future

Reply:
you are opening a huge can of worms...